src/CalendarDays/CalendarDays.js

```diff
@@ -49,7 +42,12 @@ class CalendarDays extends Component {
       }
 
       squares.push(
-        <CalendarDay day={i} startingDayRecord={startingDayRecord} key={i} />
+        <CalendarDay
+              day={i}
+              month={this.props.month}
+              id={this.props.month + i}
```

Comment:
It's best not to use proptypes that can be confused for something else. E.g., don't call a propType `className`, or in your case, `id`, since that can confused for the CSS property